test/components/viral/pill_component_test.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'view_component_test_case'
+
+module Viral
+  class PillComponentTest < ViewComponentTestCase
+    test 'default' do
+      render_preview(:default)
+
+      assert_selector 'span.bg-blue-100'
```

Comment:
**nitpick:** Can you update this to be: `assert_selector 'span.bg-blue-100', text: 'This is blue'`, that way you know the text that you are getting is within the selector.  You could also just move the `rounded-full` into each assert

Reply:
I actually hadn't seen this before fixing up the tests, so glad we were on the same page!

I've refactored the test and also added a `with_classes` test that includes the pills with added classes.

These can be found in commits 19f2995c11e145e34ba97df0db471b627bbeb7aa ced54dab21a367a1e4f3b5fd248c167cf38362ac 3fbee4acb627b1d20eef951c3bd70722aaca2af4

Reply:
Looks great, thanks